docs/troubleshooting.md

```diff
@@ -19,3 +19,18 @@ You recently downloaded or updated XCode.
 2. Choose `Legacy Build System`
 3. Remove `node_modules/react-native/third-party` and
    `node_modules/react-native/third-party-podspecs`
+
+## On iOS, authorizeAsync throws error "RNReaderSDKAuthorization is undefined"
+
+### The problem
+
+libRNReaderSDK.a is not added to your project's `Build Phases > Link Binary With Libraries`
+
+### Likely cause
+
+You failed to run `react-native link react-native-square-reader-sdk` or you are using cocoapods to link libraries.
```

Comment:
"cocoapods" --> "CocoaPods". We should honor the preferred spelling of other folks' products.